dust_extinction/averages.py

```diff
@@ -1056,3 +1057,103 @@ def evaluate(self, in_x):
         f = interp1d(self.obsdata_x, self.obsdata_axav)
 
         return f(x)
+
+class B1992(BaseExtAveModel):
+    """
+    Bastiaansen (1992) Optical Extinction Curve
+
+    Parameters
+    ----------
+    None
+
+    Raises
+    ------
+    None
+
+    Notes
+    -----
+    From Bastiaansen (1992, A&AS, 93, 449-462)
+
+    Example showing the average curve
+
+    .. plot::
+        :include-source:
+
+        import numpy as np
+        import matplotlib.pyplot as plt
+        import astropy.units as u
+
+        from dust_extinction.averages import B1992
+
+        fig, ax = plt.subplots()
+
+        # define the extinction model
+        ext_model = B1992()
+
+        # generate the curves and plot them
+        x = np.arange(1.0/ext_model.x_range[1], 1.0/ext_model.x_range[0], 0.1) * u.micron
+
+        ax.plot(x,ext_model(x),label='B1992')
+        ax.plot(1.0/ext_model.obsdata_x, ext_model.obsdata_axav, 'ko',
+                label='obsdata')
+
+        ax.set_xlabel(r'$\lambda$ [$\mu m$]')
+        ax.set_ylabel(r'$A(x)/A(V)$')
+
+        ax.legend(loc='best')
+        plt.show()
+    """
+
+    x_range = [1.0/.7873, 1.0/.3402]
+
+    rv = 3.1 #assumed!
+
+    # fmt: off
+    obsdata_x = 1.0/np.array(
+        [0.7873, 0.7505, 0.7102, 0.6681, 0.64, 
+        0.6107, 0.5821, 0.5601, 0.5407, 0.5205,
+         0.4999, 0.4708, 0.4496, 0.4395, 0.4192,
+          0.4038, 0.3785, 0.36, 0.3493, 0.3402])
+    
+    obsdata_axav = np.array(
```

Comment:
These are not A(x)/A(V), but E(x-V)/E(0.45um-0.67um) - see section 4 of the paper.  The clue that these numbers are not A(x)/A(V) is the fact that there are negative values and this is not possible for dust extinction.  Thus, the numbers needs to be translated to A(x)/A(V) to be in the standard format for the dust_extinction package.  This can be done using this curve and the assumed R(V) = A(V)/E(B-V).  If this makes sense, great.  If not I can provide more info.

Reply:
Thanks for catching this, big oversight on my part. After working on it, I now have good agreement with the other models, so I'm relatively certain I've done it right. That said, it's been a while since I've worked with extinction, so I'm pretty rusty - I will post my though process for review:

> If we have table values for E(x-V)/E(2.2-1.5), which is equivalent to [A(x)-A(V)]/(E(2.2-1.5), we can change from a color excess to just A(x) by getting rid of that A(V), then change the normalization from E(2.2-1.5) to A(V).
> If T is the values from the table, then
> A(x)/A(V) = [T + A(V)/E(2.2-1.5)] * E(2.2-1.5)/A(V) = T * E(2.2-1.5)/A(V) + 1.
> Section 5.2 states that E(2.2-1.5) = 1 corresponds to E(B-V)=.7, and we can use this to find A(V) from R(V) and E(2.2-1.5), assuming R(V) = 3.1:
> A(V) = (3.1)*(.7)*E(2.2-1.5)
> and from there, **A(x)/A(V) = T/(3.1 * .7) + 1**

It's still possible that my approach was wrong, so don't hesitate to let me know